frontend/ui/widget/progresswidget.lua

```diff
@@ -90,8 +92,8 @@ function ProgressWidget:paintTo(bb, x, y)
         local bar_width = (my_size.w-2*self.margin_h)
         local y_pos = y + self.margin_v + self.bordersize
         local bar_height = my_size.h-2*(self.margin_v+self.bordersize)
-        for i=1, #self.ticks do
-            local tick_x = bar_width*(self.ticks[i]/self.last)
+        for i,tick in pairs(self.ticks) do
```

Comment:
Try to not switch from array iterator to `pairs()`. We had bad experiences with `pairs()` in these kind of situations, and switch a lot to `ipairs` or `for i=1, #something`.

Reply:
That one is still outstanding ;).

Reply:
And since we were talking about rounding, I realize that `tick_x` was already never rounded hee, wheee... -_-".

(At a quick glance, `bar_width` should always be an int, but the MUL against `tick/last` might change that.